source/slang/slang-ast-print.h

```diff
@@ -118,7 +120,7 @@ class ASTPrinter
 
         /// Add just the parameters from a declaration.
         /// Will output the generic parameters (if it's a generic) in <> before the parameters ()
-    void addDeclParams(const DeclRef<Decl>& declRef);
+    void addDeclParams(const DeclRef<Decl>& declRef, List<Array<Index, 2>>* outParamRange = nullptr);
```

Comment:
Might be slightly nicer to have a type like `IndexRange` that hold an index range as start/end. No biggie.

Reply:
Fixed. Added `Range<T>` to core.